### ISSUE TITLE
ReferenceError: Headers is not defined

### ISSUE THREAD

**Issue by laxadev** [CodingSans/billingo-client#14]
Hello there,

I'm trying to use this package in one of my projects which is built-in AdonisJS.
When I simply try to list all the invoices `await DocumentService.listDocument().then(res => console.log({res}))`
It always throws this error:

```
ReferenceError: Headers is not defined
    at /.../node_modules/@codingsans/billingo-client/lib/client/core/request.js:156:5
    at Generator.next (<anonymous>)
    at fulfilled (/.../node_modules/@codingsans/billingo-client/lib/client/core/request.js:5:58)
    at processTicksAndRejections (node:internal/process/task_queues:96:5)
```

Same with all the others like:

`await PartnerService.createPartner({})`

What could be the problem?

**Comment by davidzambo** [comment]
I have the same error.

**Comment by Lacka90** [comment]
Thanks for reporting the issue. It's fixed in version 2.1.0
